comps/ragas/tgi/Dockerfile

```diff
@@ -1,26 +1,26 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-FROM langchain/langchain:latest
-
-RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missing \
-    libgl1-mesa-glx \
-    libjemalloc-dev \
-    vim
-
-RUN useradd -m -s /bin/bash user && \
-    mkdir -p /home/user && \
-    chown -R user /home/user/
-
-USER user
-
-COPY comps /home/user/comps
-
-RUN pip install --no-cache-dir --upgrade pip && \
-    pip install --no-cache-dir -r /home/user/comps/ragas/tgi/requirements.txt
-
-ENV PYTHONPATH=$PYTHONPATH:/home/user
-
-WORKDIR /home/user/comps/ragas/tgi/
-
-ENTRYPOINT ["python", "llm.py"]
+FROM langchain/langchain:latest
```

Comment:
This `Dockerfile` had a lot of garbage characters and I had to run it through a linter and that's why it show all lines as green but only the last line is my actual change.